src/backend.md

```diff
@@ -630,6 +630,55 @@ For serverless projects - method 3 is preferred, but not always. When you have a
       - updateBankAccount.controller.test.ts
 ```
 
+1. One test should include one action as a call of controller or job.
+
+```typescript
+   // bad
+
+   it((caseX) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+    await postSqsJob(JobName.name, {...});
+   });
+
+   // good
+
+   it((caseX) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+   });
+
+    it((caseY) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+   });
+```
+
+1. Test assertions should not verify the results of other jobs.
```

Comment:
🧇 Only the results of the main action should be checked.
responses and all DB changes.

[example](https://github.com/OsomePteLtd/core/blob/806752bf0cd180fe697a091fa7a8537aff368540/src/jobs/ticket/handleLostClientSuccessTickets.job.test.ts#L311)
The spray job has preconditions and calls the action job N times.
The action job has its preconditions and does the main action.
The test calls the spay job and checks the results of the action job.

This increases coupling and worsens refactoring.

Reply:
other => inner?